x/upgrade/abci_test.go

```diff
@@ -216,122 +238,145 @@ func (s *TestSuite) VerifyDone(newCtx sdk.Context, name string) {
 	s.Require().NotZero(height)
 }
 
-func (s *TestSuite) VerifySet() {
+func (s *TestSuite) VerifySet(skipUpgradeHeights []int64) {
 	s.T().Log("Verify if the skip upgrade has been set")
-	s.Require().NotNil(viper.GetIntSlice(s.FlagUnsafeSkipUpgrades))
+	s.Require().Equal(s.keeper.GetSkipUpgradeHeights(), skipUpgradeHeights)
 }
 
 func (s *TestSuite) VerifyConversion(skipUpgrade []int) {
```

Comment:
Can you change all these `s.VerifyXYZ(...)` methods to `VerifyXYZ(t, ...)` functions, using a normal `* testing.T`?

Once those are changed (and the calls to them), I think the rest of the `suite.Suite` references can be removed.